### DISCUSSION TITLE
Hints for Search Results

### DISCUSSION THREAD

**Discussion by jkub6** [qutebrowser/qutebrowser#7677] [Ideas] [answered]
I have found myself a few times having many search results to iterate through to reach a specific one. Would it be difficult to implement the ability to hint/select search results that are currently visible?

**Comment by The-Compiler** [comment, marked as answer]
I don't think it would be possible at all. qutebrowser only starts the search and jumps to the next result, everything else is handled by the underlying QtWebEngine/Chromium. All it knows is the active match index and the total number of matches, but not e.g. their positions.

**Reply by jkub6** [reply]
Ah, that makes sense. I appreciate the reply!
